src/QtUtils/src/Utils/LocalizationManager.cpp

```diff
@@ -39,6 +39,7 @@ namespace Locale {
 
 const QString EN = QString::fromLatin1("en");
 const QString RU = QString::fromLatin1("ru");
+const QString ZH = QString::fromLatin1("zh");
```

Comment:
This string is used for Qt translations too. But Qt has not qm file for "zh", it has separated "zh_CN" and "zh_TW" files. But it seems to LocalizationManager is not ready for such locale formats. I'll try to fix it